ci/docker/conda-integration.dockerfile

```diff
@@ -25,32 +25,28 @@ ARG node=14
 ARG jdk=8
 ARG go=1.15
 
+# Uninstall unused space-consuming packages
+# (XXX: it would be better not to install them, but they are used by other
+#  builds which are also based on conda-cpp)
+RUN conda uninstall -q clang llvmdev valgrind
```

Comment:
Please remove this statement since it's not going to reduce the overall image size. I'm going to create a follow up to  avoid having these packages in the integration image.

Reply:
see #9905